### ISSUE TITLE
Hotfix changed SIL OFL License to Apache

### ISSUE THREAD

**Issue by alexeiva** [googlefonts/fontbakery#892]
#### 1. I ran fontbakery-check-ttf.py for [this file][1]. 
Here is the Font Info
<img width="855" alt="screen shot 2016-07-31 at 15 43 59" src="https://cloud.githubusercontent.com/assets/16634089/17278891/a51949e2-5735-11e6-9ded-9918e46f8f63.png">

```
INFO     HOTFIXED: License file LICENSE.txt exists but NameID 13 (LICENSE DESCRIPTION) value is not specified for that. Value was: "This Font Software is licensed under the SIL Open Font License, Version 1.1. This license is available with a FAQ at: http://scripts.sil.org/OFL"  
```

#### 2. Fixed ttf [here][2]
<img width="863" alt="screen shot 2016-07-31 at 15 44 51" src="https://cloud.githubusercontent.com/assets/16634089/17278899/c3b4b2e2-5735-11e6-9225-ce0dcb043ac6.png">

Result:

```
This Font Software is licensed under the Apache License, Version 2.0. This license is available with a FAQ at www.apache.org/foundation/license-faq.html
```

[1]: https://github.com/alexeiva/jura/blob/master/fonts/Jura-Medium.ttf
[2]: https://github.com/alexeiva/jura/blob/master/fonts/fb-fixed/Jura-Medium.ttf

**Comment by felipesanches** [comment]
Do you have a LICENSE.txt file in the same folder as yout TTF ?

I see none on the Jura repo, but that error message is only supposed to be triggered when such file is found.

**Comment by davelab6** [comment]
Jura is OFL according to
https://github.com/google/fonts/blob/master/ofl/jura/OFL.txt

**Comment by felipesanches** [comment]
yes, I noticed that. But the script seems to have detected the presence of a LICENSE.txt as well.

This also looks bad:
https://github.com/alexeiva/jura/blob/fdf20da117cbe291dfa80d62fa10882996154818/fontbakery-check-results.json#L123-L129

**Comment by felipesanches** [comment]
I'll run the tool locally to see if I can replicate the issue.

**Comment by felipesanches** [comment]
Even though I don't see TTF files in the root of the repo, I can see both a LICENSE.txt and an OFL.txt file in there.

https://github.com/alexeiva/jura/blob/master/LICENSE.txt

https://github.com/alexeiva/jura/blob/master/OFL.txt

@alexeiva Can you please clarify these thigs:
* which command line you used
* inside which directory were you when you invoked fontbakery?
* which files were in that directory?

**Comment by felipesanches** [comment]
The contents of LICENSE.txt in Jura's repo root describe OFL licensing, but fontbakery can potentialy think it is Apache because it currenly rely only on the file name. Perhaps we should indeed look at the license file contents, specially because "LICENSE.txt" is a pretty generic name.

**Comment by alexeiva** [comment]
I copied the TTF files in the root folder to run the fb-checks. This is a confusion between UFR folder structure(fonts/*.ttf) and fb requirements. Should all TTFs be in the root by default?

Something else I noticed: Is there something wrong with version 3.001 — why is it changed to 3.00099?
```
INFO     HOTFIXED: Font version fields. Fixes: NAMEID_VERSION_STRING from 'Version 3.001' to 'Version 3.00099;' | NAMEID_VERSION_STRING from 'Version 3.001' to 'Version 3.00099;' 
```

**Comment by alexeiva** [comment]
>which command line you used

fontbakery-check-ttf.py

**Comment by alexeiva** [comment]
Oh, wait! I had both OFL.txt and license.txt. Now it's working properly. I removed licence.txt

```
INFO     OK: Found license at '/Users/alexei/github.com/alexeiva/jura/OFL.txt'  
INFO     OK: licensing entry on name table is correctly set.  
```

**Comment by felipesanches** [comment]
aha! :-D

sounds great!

**Comment by davelab6** [comment]
> I copied the TTF files in the root folder to run the fb-checks. This is a confusion between UFR folder structure(fonts/*.ttf) and fb requirements. Should all TTFs be in the root by default?

I don't understand this; fontbakery-check-ttf runs on TTF files, and doesn't matter what directory they are in; the checker will look for a description and metadata file in the same directory, but that will be true in the google/fonts repo and not the source repo. 

> Something else I noticed: Is there something wrong with version 3.001 — why is it changed to 3.00099?

I think this is a bug so I'm re-opening the issue; 3.00099 and 3.001 are equivalent but fontTools represents 3.0001 as 3.00099 for computer science number precision reasons. Font Bakery is simply confused here.

**Comment by felipesanches** [comment]
> I don't understand this; fontbakery-check-ttf runs on TTF files, and doesn't matter what directory they are in; the checker will look for a description and metadata file in the same directory, but that will be true in the google/fonts repo and not the source repo. 

The checker also looks for the LICENSE.txt or OFL.txt files in the same directory of the TTF files and verifies if they match the name table licensing entries.

**Comment by felipesanches** [comment]
the version issue will be taken caro of in a separate issue at https://github.com/googlefonts/fontbakery/issues/896

**Comment by felipesanches** [comment]
The original title of this issue is now a bit misleading:
"Hotfix changed SIL OFL License to Apache"

The unexpected hotfix was caused by the user (Alexei) placing an incorrect licensing file in his font project folder. The hotfix was consistent with the given file. Should we stop checking for coherence between licensing files and name table licensing entries?

**Comment by davelab6** [comment]
> The checker also looks for the LICENSE.txt or OFL.txt files in the same directory of the TTF files and verifies if they match the name table licensing entries.

Right; so someone using fontbakery should know that the tool expects to find those files there, and they can either duplicate them in their repo's `/fonts` directory, or, we could have a special case for if `pwd` is `fonts` and there's a `../OFL.txt` or `../LICENSE.txt` file then use that, since its sure to be a common situation. 

> The unexpected hotfix was caused by the user (Alexei) placing an incorrect licensing file in his font project folder. The hotfix was consistent with the given file.

No, the way I see, the bug that alexei found is that if you have both LICENSE.txt and OFL.txt files, we have unexpected behaviour (the for loop breaks on the first match, which is wrong); the expected behaviour is to log an error that there are 2 license files and one has to be deleted. 

> Should we stop checking for coherence between licensing files and name table licensing entries?

Definitely not! :) 

Also, I noticed that in Alexei's image, 

![images](https://cloud.githubusercontent.com/assets/16634089/17278899/c3b4b2e2-5735-11e6-9225-ce0dcb043ac6.png)

the license was changed but the URL was not. That's a bug in the fixing routine for license text; I think the best way to solve it is run have a check that the license and url match, after the individual license and url checks.
